externals/googletest/src/configureHook.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 
-CXXFLAGS="$CVMFS_BASE_CXX_FLAGS -DGTEST_HAS_TR1_TUPLE=0 -fPIC" cmake \
-    -DCMAKE_INSTALL_PREFIX=$EXTERNALS_INSTALL_LOCATION .
+mkdir build
+cd build
+
+CXXFLAGS="$CVMFS_BASE_CXX_FLAGS -fPIC" cmake ..
```

Comment:
If I remember correctly, the `-DGTEST_HAS_TR1_TUPLE=0` was necessary for certain older platforms with broken partial C++11 implementation.

Reply:
I was guessing something like that!

Reply:
Actually I wasn't able to keep it with `GTEST_HAS_TR1_TUPLE=0` I am trying to use the flag `GTEST_USE_OWN_TR1_TUPLE=1` that  should use the google version of tuple... We should check if it is enough...

Ref googletest: https://github.com/google/googletest/tree/master/googletest#choosing-a-tr1-tuple-library
Ref googlemock: https://github.com/google/googletest/tree/master/googlemock#choosing-a-tr1-tuple-library